arsceneview/src/main/java/io/github/sceneview/ar/node/ArModelNode.kt

```diff
@@ -143,7 +145,9 @@ open class ArModelNode : ArNode {
             if (!autoAnchor || !anchor()) {
                 if (arFrame.precision(lastPoseUpdateFrame) <= poseUpdatePrecision) {
                     lastPoseUpdateFrame = arFrame
-                    onArFrameHitResult(hitTest(arFrame))
+                    if (updateUnanchoredPose) {
```

Comment:
@grassydragon I've added a flag to disable the pose update instead of misusing poseUpdatePrecision to achieve it. While a gesture is in progress the pose update is disabled and reset to its original state after the gesture ended.

Reply:
Should we move the condition one level up?